backend/app/src/main/resources/db/migration/V1__Initial_ddl_script.sql

```diff
@@ -2,10 +2,11 @@
 CREATE TABLE IF NOT EXISTS table_column_info
```

Comment:
can we discuss these columns?

Reply:
These columns are used to store the names of each column in the source_data and the process_data table as well as the UI Column Name that the Users want to see on the screen.

Reply:
Please ensure consistency in naming. In the database, we use "title," but in the code, we are using "header" or "display name."